client/retry/backoff.go

```diff
@@ -53,8 +53,8 @@ func (provider *ExponentialJitterBackoff) BackoffByAttempt(attempts int) (time.D
 		backoff = MaxBackoff
 	}
 
-	if cap := int64(provider.Cap); cap > 0 && backoff > cap {
-		backoff = cap
+	if capI64 := int64(provider.Cap); capI64 > 0 && backoff > capI64 {
```

Comment:
What linter complains about the previous code?

Reply:
redefines-builtin-id: redefinition of the built-in function cap (revive)

cap is the builtin function for checking the size/cap of array/channels.